File: java/src/main/be/steinsomers/bron_kerbosch/BronKerbosch3_gpx.java
package be.steinsomers.bron_kerbosch;

import java.util.stream.Stream;

public final class BronKerbosch3_gpx implements BronKerboschAlgorithm {
    @Override
    public Stream<int[]> explore(UndirectedGraph graph) {
        return BronKerboschOrder.explore(graph, PivotChoice.MaxDegreeLocalX);
    }
}

File: kotlin/src/main/be/steinsomers/bron_kerbosch/BronKerbosch3gpx.kt
package be.steinsomers.bron_kerbosch

class BronKerbosch3gpx : BronKerboschAlgorithm {
    override fun explore(graph: UndirectedGraph, cliqueConsumer: (IntArray) -> Unit) {
        BronKerboschDegeneracy.explore(graph, cliqueConsumer, PivotChoice.MaxDegreeLocalX)
    }
}
